bolt-sidecar/src/config/mod.rs

```diff
@@ -46,10 +46,18 @@ pub struct Opts {
     #[clap(long, env = "BOLT_SIDECAR_ENGINE_API_URL", default_value = "http://localhost:8551")]
     pub engine_api_url: Url,
     /// URL for the Constraint sidecar client to use
-    #[clap(long, env = "BOLT_SIDECAR_CONSTRAINTS_URL", default_value = "http://localhost:3030")]
-    pub constraints_url: Url,
+    #[clap(
+        long,
+        env = "BOLT_SIDECAR_CONSTRAINTS_API_URL",
+        default_value = "http://localhost:3030"
+    )]
+    pub constraints_api_url: Url,
     /// Constraint proxy server port to use
```

Comment:
Since we're touching the flags, what do you think of this description?

```suggestion
    /// The port from which the Bolt sidecar will receive Builder-API requests from the Beacon client
```